src/application-manager.d.ts

```diff
@@ -6,7 +6,7 @@ import { EventTracker } from './event-tracker';
 import { Logger } from './logger';
 import { DeviceApplicationState } from './types/state';
 
-import Images from './compose/images';
+import Images, { Images } from './compose/images';
```

Comment:
Oo

Reply:
uhh, supposed to be `Image` (this needs refactored too). It's weird that this even builds though, much less passes the tests...